### ISSUE TITLE
Keyboard events

### ISSUE THREAD

**Issue by chrisblossom** [davidkpiano/react-redux-form#78]
I do not think keyboard events are currently being listened for. With my limited manual tests, using anything other than updateOn="change" validateOn="change", pressing enter on the form does not work as expected.

https://facebook.github.io/react/docs/events.html#keyboard-events
http://stackoverflow.com/questions/31272207/to-call-onchange-event-after-pressing-enter-key

**Comment by davidkpiano** [comment]
Now in `0.8.11`, `onChange` is called when the enter key is pressed on the field. Can you please confirm that this satisfies your use-case?

**Comment by chrisblossom** [comment]
```onChange``` is not being called for me still when using ```updateOn="blur"``` and pressing enter with ```0.9.0```.

**Comment by davidkpiano** [comment]
Fixed in `0.9.4`! Test case modified to reflect your use case:

```diff
    it('should change the model upon pressing Enter', () => {
      const field = TestUtils.renderIntoDocument(
        <Provider store={store}>
          <Field
            model="test.foo"
+            updateOn="blur"
          >
            <input type="text" />
          </Field>
        </Provider>
      );
```
